wrappers/hnygrpc/grpc.go

```diff
@@ -72,16 +74,26 @@ func addFields(ctx context.Context, info *grpc.UnaryServerInfo, handler grpc.Una
 	span.AddField("handler.name", handlerName)
 	span.AddField("handler.method", info.FullMethod)
 
-	md, ok := metadata.FromIncomingContext(ctx)
+	pr, ok := peer.FromContext(ctx)
 	if ok {
-		if val, ok := md["content-type"]; ok {
-			span.AddField("request.content_type", val[0])
-		}
-		if val, ok := md[":authority"]; ok {
-			span.AddField("request.header.authority", val[0])
+		if pr.Addr != net.Addr(nil) {
```

Comment:
I actually had to do a couple of minutes of poking around in docs to realize that this was just a test that pr.Addr is not nil. (Yay Go!).  (I initially thought that net.Addr(nil) was a function call that would retrieve the local machine address or something.)
```suggestion
		// if we have an address, put it on the span
		if pr.Addr != net.Addr(nil) {
```